libs/wingsdk/src/redis/redis.ts

```diff
@@ -0,0 +1,49 @@
+import { Construct } from "constructs";
+import { fqnForType } from "../constants";
+import { App, Resource } from "../core";
+
+/**
+ * Global identifier for `Bucket`.
+ */
+export const REDIS_FQN = fqnForType("redis.Redis");
+
+/**
+ * Represents a cloud redis db.
+ *
+ * @inflight `@winglang/sdk.redis.IRedisClient`
+ */
+export abstract class Redis extends Resource {
+  /**
+   * Create a new redis.
+   * @internal
+   */
+  public static _newRedis(scope: Construct, id: string): Redis {
+    return App.of(scope).newAbstract(REDIS_FQN, scope, id);
+  }
+
+  public readonly stateful = false; // TODO: redis persistence
```

Comment:
why isn't this `true`? A redis cluster is a persistent store, no?

Reply:
It can be either depends on how you configure your redis. I think by default ElastiCache and RedisLabs disable persistence (the name ElasitCahce kind of hints this).

Reply:
TIL